docs/source/getting_started/quickstart.md

```diff
@@ -63,6 +63,33 @@ alt: SAPIEN viewer
 ---
 ```
 
+## Vectorized Environment
+
+We provide an implementation of vectorized environments (for rigid-body environments) powered by the SAPIEN RPC-based render server-client system.
```

Comment:
We provide an implementation of fast vectorized environments (for rigid-body environments) powered by the SAPIEN RPC-based render server-client system. Importantly our vectorized environments parallelizes any visual observations / rendering on the GPU, making parallel interaction of environments with visual observations extremely fast. Creating a GPU parallelized vectorized environment is simple: